integration/tests/ha_chef_server.sh

```diff
@@ -0,0 +1,65 @@
+test_name="ha_chef_server"
+test_external_services=(ha_backend)
+
+source integration/services/common
+
+_frontend1_container_name="$(service_container_name "cs1")"
+_frontend2_container_name="$(service_container_name "cs2")"
+
+do_setup() {
+    do_setup_default
+
+    echo "Installing docker"
+    hab pkg install --binlink core/docker
+    echo "Installed docker"
+}
+
+do_create_config() {
+    do_create_config_default
+    cat /services/ha_backend.toml >> $test_config_path
+
+    cat << EOF >> "$test_config_path"
+[erchef.v1.sys.data_collector]
+enabled = false
+EOF
+}
+
+do_deploy() {
+    cli_bin=$(which "chef-automate")
+
+    docker_run "${_frontend1_container_name}"
+    docker_run "${_frontend2_container_name}"
+
+    docker exec -t $_frontend1_container_name \
+        "$cli_bin" deploy config.toml \
+            --product chef-server \
+            --hartifacts "$test_hartifacts_path" \
+            --override-origin "$HAB_ORIGIN" \
+            --manifest-dir "$test_manifest_path" \
+            --admin-password chefautomate \
+            --accept-terms-and-mlsa
+
+    docker exec -t $_frontend1_container_name \
+        "$cli_bin" bootstrap bundle create -o bootstrap.abb
```

Comment:
is `-o` a required switch? :thinking: I thought we went with an optional path but I guess I'm wrong.

Reply:
it's not required. it means force overwrite if the file exists which i wrote out of habit from running it locally